lib/iron_worker_ng/feature/python/merge_requirements.rb

```diff
@@ -0,0 +1,22 @@
+module IronWorkerNG
+  module Feature
+    module Python
+      module MergeRequirements
+        module InstanceMethods
+          def merge_requirements(path)
+            feature = IronWorkerNG::Feature::Base.new(self)
+            IronWorkerNG::Fetcher.fetch_to_file(feature.rebase(path)) do |requirements|
+              specs = []
+              File.readlines(requirements).map { |line| specs << line.split('==')}
```

Comment:
Are you sure it works correctly for all possible version formats? I'm not familiar to python enough